commercial/app/views/contentapi/item.scala.html

```diff
@@ -1,8 +1,83 @@
 @(item: model.ContentType, optLogo: Option[String], optCapiTitle: Option[String], optCapiLink: Option[String], optCapiAbout: Option[String], optCapiButtonText: Option[String], optCapiReadMoreUrl: Option[String], optCapiReadMoreText: Option[String], optSponsorType: Option[String], optSponsorLabel: Option[String], optClickMacro: Option[String], optOmnitureId: Option[String])(implicit request: RequestHeader)
 
 @import conf.switches.Switches.NewCommercialContent
+@import views.html.fragments.inlineSvg
 
 @defining(("1_0", "2014-08-14")) { case (version, date) =>
+    @if(NewCommercialContent.isSwitchedOn) {
+    <div class="commercial commercial--dfp-single paid-content--advertisement-feature commercial--tone-paidfor" data-link-name="merchandising | capi | single @optOmnitureId">
+        <div class="commercial__inner">
+            <div class="commercial__header commercial--paidfor">
+                <div class="paidfor-meta">
+                    <span class="paidfor-meta__label">Paid Content</span>
+                    <div class="paidfor-label paidfor-meta__more has-popup">
+                        <button class="u-button-reset paidfor-label__btn popup__toggle" data-toggle="paidfor-popup--capi-component">About @inlineSvg("arrow-down", "icon")</button>
+                        <div class="popup is-off paidfor-popup paidfor-popup--capi-component">
+                            <h3 class="paidfor-popup__title">Paid stories are paid for and approved by an advertiser and created by the Guardian Labs commercial editorial team</h3>
```

Comment:
@regiskuckaertz I've updated the text and created card so we should look at it one more time to possibly create fragment for this as it is across multiple components as you said.